.github/workflows/escrow.yml

```
@@ -77,6 +77,7 @@ jobs:
           build-args: |
             REACT_APP_NODE_ADDRESS=${{ secrets.REACT_APP_NODE_ADDRESS }}
             REACT_APP_CODE_ADDRESS=${{ secrets.REACT_APP_CODE_ADDRESS_ESCROW }}
+            REACT_SENTRY_DSN=${{ secrets.REACT_SENTRY_DSN_ESCROW }}
```

```suggestion
            REACT_APP_SENTRY_DSN=${{ secrets.REACT_SENTRY_DSN_ESCROW }}
```

`create-react-app` envs should be prefixed with `REACT_APP`.